en/docs/deploy-operate/deploy/serverless-deployment.md

```
@@ -6,159 +6,261 @@ description: Deploy integrations as serverless functions on AWS Lambda and Azure
   azure_functions/
-    my_integration/
+    <function-name>/
       function.json
-      handler.jar
     host.json
 ```
 
-### Step 3 -- deploy to Azure
+The build output lists the functions and prints the commands to run locally and deploy:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)